code/drasil-example/Drasil/DblPendulum/Assumptions.hs

```diff
@@ -20,18 +22,18 @@ yAxisDir        = cic "yAxisDir"          yAxisDirDesc        "yAxisDir"
 startOrigin     = cic "startOrigin"       startOriginDesc     "startOrigin"     assumpDom
 
 pend2DMotionDesc :: Sentence
-pend2DMotionDesc = S "The" +:+ phrase pendulum +:+ phrase motion `S.sIs` phrase twoD +:+. sParen (getAcc twoD)
+pend2DMotionDesc = atStartNP (the (compoundNC pendulum motion)) `S.sIs` phrase twoD +:+. sParen (getAcc twoD)
```

Comment:
it's probably not a good idea to use `compoundNC` inline like this. You should give this a Haskell-level name.